### ISSUE TITLE
DynamoDb transact write items errors for Update expression multiple set action

### ISSUE THREAD

**Issue by SharnikaNadarajah** [getmoto/moto#7817]
When using moto mock dynamodb, transact_write_items operation errors even when there is only a single SET action in an update action to dynamo. Error as below

_**botocore.exceptions.ClientError: An error occurred (ValidationException) when calling the TransactWriteItems operation: Invalid UpdateExpression: The "SET" section can only be used once in an update expression;**_

Issue was found in moto 5.0.10.
Reverting back to 5.0.9 works as expected.

**Comment by bblommers** [comment]
Hi @SharnikaNadarajah, thanks for raising this. Can you share an example test on how to reproduce this? Because I wasn't able to.

The below invocation for example works fine for me:
```
ddb_client.transact_write_items(
        TransactItems=[
            {
                "Update": {
                    "TableName": table_name,
                    "Key": {"pk": {"S": "test"}},
                    "UpdateExpression": "SET #xxx = :xxx",
                    "ExpressionAttributeNames": {"#xxx": "xxx"},
                    "ExpressionAttributeValues": {":xxx": {"S": "456"}},
                }
            }
        ]
    )
```

(As an FYI, there is a [confirmed bug](https://github.com/getmoto/moto/issues/7819) when you try to execute this operation without any `ExpressionAttributeName`-s, that will be fixed separately.)

**Comment by bblommers** [comment]
OK, so the problem was that two actions are allowed, just not multiple clauses. I.e.: `SET a=b, x=y` is allowed, but `SET a=b SET x=y` is not.

This is now fixed in `moto >= 5.0.11.dev17` - thanks for raising this!
